### DISCUSSION TITLE
Fake Finder

### DISCUSSION THREAD

**Discussion by ksaffarian** [WeAreMahsaAmini/Core#157] [Ideas]
One of the biggest issues that we always suffer from is fake people, fake news, fake missions, fake gathering, fake whatever.

We can easily get distracted with fake content and start advocating and endorsing something that is not real, not correct and not legit.

If we could have a solution, like a simple mobile app that people can quickly nominate fake stuff and vote over them, we can potentialy save lives, money and time and focus on our goals and mission with better efficiency.

**Comment by Setareh4IRAN** [comment]
Good idea, but who is responsible for finding out which news is real and which fake? If we want to create poll for people to vote , this is the exact goal of fake news! distracting the majority of people , right?

**Reply by ksaffarian** [reply]
Sure, this issue that you mentioned always can happen, unless you have ppl that you pay to investigate and do fact checking.
This idea works in the same way that for example your phone marks a call as spam. It is based on crowd decision.
IMO, even if it prevents a single fake stuff from surfing the internet, there is a win in this system.
Maybe we can find some ways of increasing the accuracy, based on some AI models, twitter crawl, etc.

**Comment by blackwolf202** [comment]
Seems like to be a duplicated idea https://github.com/WeAreMahsaAmini/Core/discussions/9

**Reply by ksaffarian** [reply]
No, #9 only focuses on bot detection. This idea covers wider scope and focuses on content, rather tha  the publisher.
